lib/vagrant-ovirt4/action/create_network_interfaces.rb

```diff
@@ -27,8 +27,18 @@ def call(env)
             iface_options = scoped_hash_override(options, :ovirt)
           end
 
+          config = env[:machine].provider_config
+          @logger.info("Finding network #{iface_options[:network_name]} for given cluster #{config.cluster}")
+          network_profile_id = nil
+          clusters_service = env[:connection].system_service.clusters_service
+          cluster = clusters_service.list(search: "name=#{config.cluster}").first
```

Comment:
I might opt to keep this and change your PR.
Right now I don't distinguish the network per cluster, probably because I forgot.

So if you had a network `ovirtmgmt` defined in cluster `a` and `ovirtmgmt` defined in cluster `b` i'm actually not sure what would happen given my current commit in master